specific requirements

Deco1985 opened the pull request:
**Requirements definition and specification.**
The _"use case"_ format was used to specify requirements. Still need to review small details, I hope we have a meeting, thx guys.

Deco1985 committed:
specific requirements

JosuehCA commented on Requerimientos.md:
```diff
@@ -1,53 +1,348 @@
+
+      - El usuario dispondrá de un notificador de reportes y alertas que permite obtener información valiosa sobre el estado del almacén, esto para evitar el desperdicio o merma.
+
+### **RF-05**: Etiquetado apartir de "Lotificación y Trazabilidad"
```
Modificar definición. Párrafo deshilvanado y un error ortográfico

JosuehCA replied:
Error ortográfico

JosuehCA commented on Requerimientos.md:
Faltas de ortografía

JosuehCA commented on Requerimientos.md:
Faltas de ortografía.

JosuehCA commented on Requerimientos.md:
Faltas de ortografía y cohesión.

JosuehCA commented on Requerimientos.md:
```diff
@@ -1,53 +1,348 @@
+      - El usuario dispondrá de un notificador de reportes y alertas que permite obtener información valiosa sobre el estado del almacén, esto para evitar el desperdicio o merma.
+
+### **RF-05**: Etiquetado apartir de "Lotificación y Trazabilidad"
+      - Los datos de nuestro programa se ordenarán a partir del sistema implementado por nuestro cliente, el sistema "Lote". La clasificación lote se rige por la trazabilidad, en resumen, es la información de cada embarque de pescado, de donde vino, quien lo pescó, forma de empaque, etc. 
```
Modificar definición y cohesión de palabras

JosuehCA commented on Requerimientos.md:
Error de ortografía.

JosuehCA commented on Requerimientos.md:
Error de redacción. Coma en vez de punto, y seguir la idea desde ahí

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Errores ortográficos.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Párrafo extra. Eliminar y actualizar conteo.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Error ortográfico y de redacción.

JosuehCA commented on Requerimientos.md:
Error de acentuación.

JosuehCA commented on Requerimientos.md:
Error ortográfico y de acentuación.

JosuehCA commented on Requerimientos.md:
Error de acentuación.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Error de redacción.

JosuehCA commented on Requerimientos.md:
Error ortográfico y de acentuación.

JosuehCA commented on Requerimientos.md:
Errores ortográficos.

JosuehCA commented on Requerimientos.md:
Errores de redacción y cohesión.

JosuehCA commented on Requerimientos.md:
Error de acentuación.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Error de cohesión.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Errores ortográficos.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Errores ortográficos.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Error de redacción en "despliegará" -> "desplegará"

JosuehCA commented on Requerimientos.md:
No se comenzó con mayúscula.

JosuehCA commented on Requerimientos.md:
Error ortográfico.

JosuehCA commented on Requerimientos.md:
Errores ortográficos y de redacción.

JosuehCA commented on Requerimientos.md:
Error de acentuación y redacción.

JosuehCA commented on Requerimientos.md:
"Litificado"?

JosuehCA commented on Requerimientos.md:
Errores de acentuación y redacción.

JosuehCA reviewed (changes requested):
Corrige todos los errores que se indican en los comentarios, y cambia el diseño del MD de la sección de requisitos. La información no se aprecia claramente. Contempla modificar RF-05.

Deco1985 committed:
Update Requerimientos.md

Errores gramaticales corregidos.
Cambio en el requemiento RF - 05

Deco1985 committed:
Update Requerimientos.md

Deco1985 committed:
Update Requerimientos.md

Last changes

JosuehCA committed:
Update 2 Requerimientos

No cambiaste el formato dentro del MD para que se visualice correctamente. Ya se realizó el cambio.

JosuehCA reviewed (approved):
Bien. Mejora tu ortografía. Se modificó el impacto y la urgencia del RF-04

johanaguilar12 committed:
Update Requerimientos.md

Cambios en la redacción. Se modificaron algunas prioridades en el Impacto y Urgencia de los requisitos.

johanaguilar12 committed:
Update Requerimientos.md

Cambios en los Requisitos No Funcionales.

RicardoCv0 committed:
Update Requerimientos.md

Cambio en la Urgencia de Alertas.

RicardoCv0 reviewed (approved):
Buen trabajo
